zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/GroupDeleteProcessor.java

```diff
@@ -118,20 +119,20 @@ final var record = command.getValue();
   }
 
   private void removeAssignedEntities(final GroupRecord record) {
-    final var groupKey = record.getGroupKey();
```

Comment:
🔧 I just noticed that we forgot to change the error message in the distributed command flow. This still sets the group key in the message instead of the id. This should be fixed.